langserver/internal/refs/refs_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"strings"
 	"testing"
 
-	_ "gopkg.in/inconshreveable/log15.v2"
+	_ "github.com/inconshreveable/log15"
```

Comment:
The `init` for log15 doesn't appear to do anything relevant—is this import even needed?